docs/src/pages/components/checkboxes/CheckboxesGroup.tsx

```diff
@@ -38,7 +38,11 @@ export default function CheckboxesGroup() {
 
   return (
     <div className={classes.root}>
-      <FormControl component="fieldset" className={classes.formControl}>
+      <FormControl
+        component="fieldset"
+        className={classes.formControl}
+        variant="standard"
```

Comment:
Does it have an impact on checkboxes? I would be surprised if it has one

Reply:
It impacts the `FormHelperText`.